### ISSUE TITLE
Model v0.9.1 consumes much more VRAM

### ISSUE THREAD

**Issue by Kvento** [Lightricks/ComfyUI-LTXVideo#81]
It seems that my new model consumes much more VRAM than model v0.9.
On model v0.9 I could generate 1280x720 with 97 frames and all this fit in 11gb VRAM (on 2080ti).
And on model v0.9.1 I could only start generation with a resolution of 632x280 at 97 frames. At the same time, during generation there is still free VRAM, but during VAE decoding, the consumption increases sharply. So I was able to achieve only this resolution taking into account the consumption of the VAE decoder.

I am using the new workflow from the description.

The question is whether this is related to the model or to the nodes?

**Comment by gshawn3** [comment]
I'm seeing the same issue here as well, VRAM usage is much higher on 0.9.1 than it was on 0.9. I keep running into Out Of Memory errors with the provided i2v workflow. I don't have a solution for this, unfortunately, but wanted to point out this is not an isolated issue as it was also reported here: https://www.reddit.com/r/StableDiffusion/comments/1hhz17h/comment/m2v4hi0/

**Comment by apedance** [comment]
> Also same issue here, but I was able to fix it with a Clean VRAM node right after the Sampler and before VAE Decode. I also put another one between the Guider and Sampler. Fixed the issue for me.

can you provide a screenshot, please?

**Comment by jroc22** [comment]
> > Also same issue here, but I was able to fix it with a Clean VRAM node right after the Sampler and before VAE Decode. I also put another one between the Guider and Sampler. Fixed the issue for me.
> 
> can you provide a screenshot, please?

![Screenshot 2024-12-20 115622](https://github.com/user-attachments/assets/1275518c-e0c9-48e5-916c-fabb5e58fdf2)

**Comment by YAY-3M-TA3** [comment]
> > > Also same issue here, but I was able to fix it with a Clean VRAM node right after the Sampler and before VAE Decode. I also put another one between the Guider and Sampler. Fixed the issue for me.
> > 
> > 
> > can you provide a screenshot, please?
> 
> ![Screenshot 2024-12-20 115622](https://private-user-images.githubusercontent.com/110642086/397850596-1275518c-e0c9-48e5-916c-fabb5e58fdf2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ3NjYwNzgsIm5iZiI6MTczNDc2NTc3OCwicGF0aCI6Ii8xMTA2NDIwODYvMzk3ODUwNTk2LTEyNzU1MThjLWUwYzktNDhlNS05MTZjLWZhYmI1ZTU4ZmRmMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyMVQwNzIyNThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNjE4ZjQ0NmZjYzUxNzYwMGMzZmFkMjAzODQ5OTFlYWI5NzY3ZTFlOTIwMzJhZDQ1NGI2MTBlNTQ4MjMyZGE3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Cw69sFQaxmqA0X8KEIA3QeYh2CBp3p5SBLmKPIkRBvs)

Where do you find that Clean VRAM node from - I don't see it anywhere?  is it a part of another package?

**Comment by nuxs71** [comment]
You can use the free memory tool as well to clean ram and/or vram. Here the link:
[https://github.com/ShmuelRonen/ComfyUI-FreeMemory]
(To install open cmd from the comfyui node directory and git clone it!) 😎

**Comment by apedance** [comment]
Even with Clean VRAM I can't get this workflow to run properly.
I get to the SamplerStage with the Clean VRAM nodes but it seems to get stuck there.
RTX 2080 ti 11gb vram - whole workflow seems to consume about 14gb vram.

![image](https://github.com/user-attachments/assets/3c6cce88-25f9-4934-9b96-a163372f4ea0)
![image](https://github.com/user-attachments/assets/60f48ea3-21a5-4717-a778-34cbddd3c964)

**Comment by Kvento** [comment]
> Even with Clean VRAM I can't get this workflow to run properly. I get to the SamplerStage with the Clean VRAM nodes but it seems to get stuck there. RTX 2080 ti 11gb vram - whole workflow seems to consume about 14gb vram. might be even more if I could get further. Does Florence2 take up vram?

Clean VRAM does not reduce the memory consumption of the model during generation. It simply allows us to get rid of the increased consumption during VAE decoding. I believe that the model itself has become much more memory-consuming than the previous one.
All that remains is to wait until someone releases a quantized GGUF version. Until then, with the amount of VRAM we have, we can either make do with low-resolution generation or use the old model.

**Comment by YAY-3M-TA3** [comment]
Yeah, the new VAE node might need to expose a tiling and minimum size - then it can fit in VRAM...

**Comment by Packsod** [comment]
![image](https://github.com/user-attachments/assets/7bc9b1ac-b94c-4a9e-bfdc-42c683774c36)
Because the official workflow of LTXVideo does not support ComfyUI's VAE Decode (Tiled) node, otherwise it will save a lot of vram.

In addition, after the generation is completed, vram should also be cleared before the decoding stage. I think this is not a problem with the model itself, but a defect of this node group.

**Comment by x4080** [comment]
do we need to use the new VAE from lighttricks repo ? I dont use the new VAE file, it works
